core/src/com/ericc/the/game/MainGame.java

```diff
@@ -29,34 +30,42 @@
 
     private Engines engines = new Engines();
 
+    public final static boolean DEBUG = true; ///< turns the debug mode on and off
```

Comment:
to powyrzucać do pliku konfiguracyjnego, ale to jest poza scopem tego PR, założyłem stosowne issue żeby zrobić to potem